docs/development/how_to_use_dataset.md

```diff
@@ -118,8 +111,8 @@ api group: 'org.apache.commons', name: 'commons-csv', version: '1.7'
 In order to extend the dataset, the following dependencies are required:
 
 ```
-api "ai.djl:api:0.8.0"
-api "ai.djl:basicdataset:0.8.0"
+api "ai.djl:api:0.9.0"
```

Comment:
0.9.0 has not been released, we keep released version in README file.

Reply:
Thanks for the info, I’ll revert the version.